projects/cgif/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2023 Google Inc.
+# Copyright 2025 Google Inc.
```

Comment:
We prefer to keep the original year in the copyright, could you do this?

Reply:
2738c3f750e47c8b8abcc9fe53d6250f1750a859